imgix/drift
# feat(mobile): add touch delay to allow scroll on mobile

# Description
This is a fix for touch capable devices, there drift zoom prevents scrolling while displayed on screen. The Drift Stage cant be use for start oft gestures, because the event will always be 
default-prevented. Its not breaking because you need to set touchDelay (> 0) for this to work. IF there is touchdelay set, drift only prevents browsers default behaviour if the zoom is active. otherwise the events will pass.  Fixes #77

# Bug Fix

# Steps to Test:
- Create an (large) image which has drift zoom applied on.
- Configure drift with no touchDelay.
- Add some "lorem ipsum" text below.
- Use a touch capable mobile (iphone X here)
- open the page on a mobile an make sure the image covers the complete display
- try to scroll down
You cant!

With fix:
- set touchDelay to 500
- try to scroll down (while gesture starts on the image)
You can scroll now!

- holding you finger on the image on the image for > 500ms
now you can move around the zoom maginfication bubble without scrolling
